packages/wdio-selenium-standalone-service/src/launcher.ts

```diff
@@ -48,11 +48,11 @@ export default class SeleniumStandaloneLauncher {
         this.skipSeleniumInstall = Boolean(options.skipSeleniumInstall)
     }
 
-    async onPrepare(config: Config) :Promise<void> {
+    async onPrepare(config: Config): Promise<void> {
         this.watchMode = Boolean(config.watch)
 
         if (!this.skipSeleniumInstall) {
-            await promisify(SeleniumStandalone.install)(this.installArgs)
+            await promisify(SeleniumStandalone.install.bind(this.installArgs))
```

Comment:
could someone help me on what should be used here.

Reply:
I think it should not be changed

Reply:
If not changed, then i am getting the below error:

`(property) SeleniumStandaloneLauncher.installArgs: Partial<SeleniumStandalone.InstallOpts> Expected 0 arguments, but got 1.ts(2554)`

Reply:
you can define `installSeleniumStandalone = promisify(SeleniumStandalone.install)`
it should have the same type as `SeleniumStandalone.install` but result should be wrapped with `Promise<>`